mslib/mscolab/utils.py

```diff
@@ -64,9 +64,9 @@ def os_fs_create_dir(dir):
         try:
             _ = fs.open_fs(dir)
         except fs.errors.CreateFailed:
-            logging.error(f'Make sure that the FS url "{dir}" exists')
+            logging.error('Make sure that the FS url "{0}" exists'.format(dir))
```

Comment:
always %s syntax (old string formatting) not .format() that has the same problem as f'strings for logging